packages/zowe-explorer/package.json

```diff
@@ -121,6 +121,21 @@
       }
     ],
     "commands": [
+      {
+        "command": "zowe.jobs.sortbyreturncode",
+        "title": "Sort by returncode",
```

Comment:
Please add these new command titles to the package.nls.json file and reference them here like the other commands for localization, thanks.

Reply:
Modified the command titles in package.json and updated the respective in package.nls.json file